MeshLib/MeshEditing/MeshRevision.cpp

```diff
@@ -253,24 +305,30 @@ void MeshRevision::reduceElement(MeshLib::Element const*const element,
 	/***************
 	 * TODO: modify neighbouring elements if one elements has been subdivided
 	 ***************/
-	if (element->getGeomType() == MeshElemType::TRIANGLE && min_elem_dim == 1)
+	if (element->getGeomType() == MeshElemType::TRIANGLE && min_elem_dim == 1) {
```

Comment:
everywhere else in this class the ```{``` is on a new line ... could you please do this here, too, otherwise it'll be easily overlooked.